metadata/licenses/defs.bzl

```diff
@@ -0,0 +1,8 @@
+"""Public API of `@package_metadata//licenses`."""
+
+load("//licenses/private/rules:license.bzl", _license = "license")
+
+# Providers.
+
+# Rules
+license = _license
```

Comment:
I believe the new convention is to avoid a `defs.bzl` file and instead have each rule be in its own `bzl` file. eg. `//metadata/liceneses/rules:license.bzl` would contain the `license` rule.

Reply:
Done